__tests__/components/AccountAddressDisplay.spec.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * (C) Symbol Contributors 2021
```

Comment:
2022 now. @Wayonb can we grab some CI to automatically bump the year of our header files? ^^